vignettes/getting-started-with-teal.Rmd

```diff
@@ -35,75 +34,84 @@ library(teal)
 
 app <- init(
   data = teal_data(IRIS = iris, MTCARS = mtcars),
-  modules = example_module(),
-  header = "My first teal application"
+  modules = modules(
+    example_module("Module 1"),
+    example_module("Module 2")
```

Comment:
I thought it was intentional. The first wireframe is just a template that shows what is possible. And the second one is just an example.
This also shows that people can have any number of modules as they like when we show three in the first image and two in next.

Reply:
Oh my 🤦🏽  Let me add that. Thanks!

Reply:
Oh my 🤦🏽 Let me add that. Thanks!

Reply:
No wait. I'm getting confused again. I think it's okay to have just two modules in the "example wireframe"
And have three modules in the "template wireframe" because of my reasoning above. What are your thoughts?

Reply:
I'm fine with the reasoning that you provided earlier.